release-notes/3.0/preview/api-diff/preview5/3.0-preview5_System.Runtime.Intrinsics.X86.md

```diff
@@ -0,0 +1,293 @@
+
++        public static Vector128<double> CompareScalarEqual(Vector128<double> left, Vector128<double> right);
++        public static Vector128<double> CompareScalarGreaterThan(Vector128<double> left, Vector128<double> right);
++        public static Vector128<double> CompareScalarGreaterThanOrEqual(Vector128<double> left, Vector128<double> right);
++        public static Vector128<double> CompareScalarLessThan(Vector128<double> left, Vector128<double> right);
++        public static Vector128<double> CompareScalarLessThanOrEqual(Vector128<double> left, Vector128<double> right);
++        public static Vector128<double> CompareScalarNotEqual(Vector128<double> left, Vector128<double> right);
++        public static Vector128<double> CompareScalarNotGreaterThan(Vector128<double> left, Vector128<double> right);
++        public static Vector128<double> CompareScalarNotGreaterThanOrEqual(Vector128<double> left, Vector128<double> right);
++        public static Vector128<double> CompareScalarNotLessThan(Vector128<double> left, Vector128<double> right);
++        public static Vector128<double> CompareScalarNotLessThanOrEqual(Vector128<double> left, Vector128<double> right);
++        public static Vector128<double> CompareScalarOrdered(Vector128<double> left, Vector128<double> right);
++        public static bool CompareScalarOrderedEqual(Vector128<double> left, Vector128<double> right);
++        public static bool CompareScalarOrderedGreaterThan(Vector128<double> left, Vector128<double> right);
++        public static bool CompareScalarOrderedGreaterThanOrEqual(Vector128<double> left, Vector128<double> right);
++        public static bool CompareScalarOrderedLessThan(Vector128<double> left, Vector128<double> right);
++        public static bool CompareScalarOrderedLessThanOrEqual(Vector128<double> left, Vector128<double> right);
++        public static bool CompareScalarOrderedNotEqual(Vector128<double> left, Vector128<double> right);
++        public static Vector128<double> CompareScalarUnordered(Vector128<double> left, Vector128<double> right);
++        public static bool CompareScalarUnorderedEqual(Vector128<double> left, Vector128<double> right);
++        public static bool CompareScalarUnorderedGreaterThan(Vector128<double> left, Vector128<double> right);
++        public static bool CompareScalarUnorderedGreaterThanOrEqual(Vector128<double> left, Vector128<double> right);
++        public static bool CompareScalarUnorderedLessThan(Vector128<double> left, Vector128<double> right);
++        public static bool CompareScalarUnorderedLessThanOrEqual(Vector128<double> left, Vector128<double> right);
++        public static bool CompareScalarUnorderedNotEqual(Vector128<double> left, Vector128<double> right);
+-        public static Vector128<double> CompareUnorderedScalar(Vector128<double> left, Vector128<double> right);
+
+-        public unsafe static void StoreLow(long* address, Vector128<long> source);
```

Comment:
For reference, this was renamed to `StoreScalar` since there is no `StoreHigh` counterpart. It makes it more consistent with the other API names.